webui/src/Surfaces/KnownSurfacesTable.tsx

```diff
@@ -239,7 +239,7 @@ const SurfaceRow = observer(function SurfaceRow({
 			onClick={handleSurfaceClick}
 		>
 			<div className="grid-cell">{index !== undefined ? `#${index}` : ''}</div>
-			<div className="grid-cell">
+			<div className="grid-cell" style={{ paddingLeft: index === undefined ? '2em' : 'default' }}>
```

Comment:
It would be nicer for this to use the I think `ps-4` (or some other number) class for this.
Something like:
```
className={classNames('grid-cell', {
	'ps-4': index === undefined,
})}
```
(a few lines above is another example of this approach)

Reply:
Thank you for your guidance @Julusian !!
Done.